lib/Braintree/Http.php

```diff
@@ -131,6 +131,11 @@ public function _doUrlRequest($httpVerb, $url, $requestBody = null)
             curl_setopt($curl, CURLOPT_POSTFIELDS, $requestBody);
         }
 
+        if (!empty($this->_config->getProxy())) {
```

Comment:
Prior to PHP version 5.5 this will result in the following error:
**Can't use method return value in write context**